flake.nix

```diff
@@ -195,10 +195,20 @@
       }) pkgsContext;
 
       # ------------------------------------------------------------------------ #
+
       devShells = builtins.mapAttrs (system: pkgsBase: {
         default = pkgsBase.floxDevelopmentPackages.callPackage ./shells/default { };
       }) pkgsContext;
-    }; # End `outputs'
+
+      # ------------------------------------------------------------------------ #
+
+      # NixOS/Darwin/HomeManager module
+      nixosModules.flox = import ./modules/nixos.nix overlays.default inputs.nixpkgs;
+      darwinModules.flox = import ./modules/darwin.nix overlays.default inputs.nixpkgs;
+      homeModules.flox = import ./modules/home.nix overlays.default inputs.nixpkgs;
```

Comment:
question nonblocking: my overlays knowledge is weak, but why don't you pass pkgsContext through here rather than overlays.default and inputs.nixpkgs?

Reply:
Thank you. I completely overlooked `pkgsContext`.